app/models/recoverable_revision.rb

```diff
@@ -28,7 +28,7 @@ def self.all
       versions = PaperTrail::Version.where(event: 'destroy', item_type: model.to_s).
         joins("LEFT JOIN #{table_name} ON item_id=#{table_name}.id").
         where("#{table_name}.id IS NULL"). # avoid showing deleted objects
-        order(created_at: :desc)
+        order(id: :desc)
```

Comment:
I understand why this change is being made, but I think it's worth adding a comment to explain why.